lib/core/bootstrap.js

```diff
@@ -128,7 +128,7 @@ module.exports = async function () {
     JSON.stringify(
       {
         pack: {
-          pack_format: 6,
+          pack_format: 7,
```

Comment:
pack format will be atleast 8 in 1.18, prob time to add an option for it :(

you don't need to if you dont want but its prob a good idea.

Reply:
This is a good suggestion. I'll definitely check it out